heltour/tournament/admin.py

```diff
@@ -661,7 +661,9 @@ def player_info_view(self, request, object_id, player_name):
             'league': season.league,
             'player': season_player.player,
             'reg': reg,
-            'has_played_20_games': has_played_20_games
+            'has_played_20_games': has_played_20_games,
+            'date_created': sp.registration.friends if sp.registration else None,
```

Comment:
Also, I think this is the wrong spot to add this. You probably want line 610, in which case `sp` would be defined.